en.txt

```diff
@@ -0,0 +1,67 @@
+About me:
+	I mostly have experience with Linux servers, Unix, c programming language
+	I like Implementing interesting ideas.
+	I have worked briefly in 2 startups, here is what I did:
+	Start-up 1 - inn0web, website: https://inn0web.com, (time period 2022.08 - 2022.10):
+		- I wrote a bot for finding bicycles of a specific price, in particular locations.
+		When found, send information to Telegram.
+		- The bot is designed to find the most popular Reddit posts, and according to the topic, make 10 min
+		video montage to upload them to different YouTube channels
+	Start-up 2 - Monopulse, website: https://www.monopulse.eu, (time period ~2023.02): 
+		- I was assigned a task to work with embedded Linux in the yacto project.	
+	~2023.09 I was selling WordPress installation services. 
```

Comment:
you are not seller. Yo are installer. For example: 2023.01 - 2023.09 I've installed WordPress about 8 projects with different complexity;